atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -101,9 +101,21 @@ class ScriptExecutionCallback : public blink::WebScriptExecutionCallback {
 
   void Completed(
       const blink::WebVector<v8::Local<v8::Value>>& result) override {
-    if (!result.empty() && !result[0].IsEmpty())
-      // Right now only single results per frame is supported.
-      promise_.Resolve(result[0]);
+    if (!result.empty()) {
+      if (!result[0].IsEmpty()) {
+        // Right now only single results per frame is supported.
+        promise_.Resolve(result[0]);
+      } else {
+        promise_.RejectWithErrorMessage(
+            "Script failed to execute, this normally means an error "
+            "was thrown. Check the renderer console for the error."
+            "was thrown, check the renderer console for the error");
```

Comment:
This is here twice.

Reply:
How strange 😕 I definitely wouldn't type the same thing twice, the auto-formatter might have done this 🤔

Reply:
i think it's because you directly committed my suggestion in the github UI, which can't handle suggesting changing multiple lines